index.md

```diff
@@ -5,43 +5,43 @@ layout: base
 # Features
 ### Section description
 
+Dune is a build system for OCaml projects.
+Using it, you can build executables, libraries, run tests, and much more.
+
 <div class="flex-grid">
 <div class="col text-container">
-#### Great thing
-Dune does amazing things that will change your life. You will
-wonder what you did without it and thank the OCaml wizard every
-day.
+#### Dune knows OCaml and its ecosystem
+Dune has rules that precisely capture how the different compilers work.
```

Comment:
When I read this sentence it made me think that Dune supports several different languages. What compilers are you thinking of?

Reply:
`ocamlc` and `ocamlopt`, but yes that's confusing. I'll rework that part.